### PR TITLE
Update package-management.md

### PR DESCRIPTION
**Pull request by mirnawong1**
Added a disclaimer that SSH key for private packages, can only be used for the CLI and not dbt Cloud. This is listed in line 148 in parenthesis and per slack thread here: https://dbt-labs.slack.com/archives/C017GDLAF7D/p1654602574639929

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [x] I've added versioning components, as described in ["Versioning Docs"](../contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](../website/docs/docs/guides/migration-guide)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

**Comment by jeremyyeo** [comment]
Relevant: https://github.com/dbt-labs/docs.getdbt.com/pull/1201

File: website/docs/docs/building-a-dbt-project/package-management.md [hunk 1]
```diff
@@ -158,6 +160,9 @@ packages:
 
 </File>
 
+If you're using dbt Cloud, the SSH key method will not work, but you can use the [HTTPS Git Token Method](https://docs.getdbt.com/docs/building-a-dbt-project/package-management#git-token-method).
```
Comment (runleonarun) [inline website/docs/docs/building-a-dbt-project/package-management.md:163]: I recommend using a relative link:
```suggestion
If you're using dbt Cloud, the SSH key method will not work, but you can use the [HTTPS Git Token Method](/docs/building-a-dbt-project/package-management#git-token-method).
```

File: website/docs/docs/building-a-dbt-project/package-management.md [hunk 2]
```diff
@@ -145,8 +145,10 @@ As of v0.14.0, dbt will warn you if you install a package using the `git` syntax
 
 ### Private packages
 
-#### SSH Key Method
-Private packages can be cloned via SSH and an SSH key. When you use SSH keys to authenticate to your git remote server, you don’t need to supply your username and password each time. Read more about SSH keys, how to generate them, and how to add them to your git provider here: [Github](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh) and [GitLab](https://docs.gitlab.com/ee/ssh/).
+#### SSH Key Method (Command Line only)
+If you're using the Command Line, private packages can be cloned via SSH and an SSH key. 
+
+When you use SSH keys to authenticate to your git remote server, you don’t need to supply your username and password each time. Read more about SSH keys, how to generate them, and how to add them to your git provider here: [Github](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh) and [GitLab](https://docs.gitlab.com/ee/ssh/).
```
Comment (runleonarun) [inline website/docs/docs/building-a-dbt-project/package-management.md:151]:
```suggestion
When you use SSH keys to authenticate to your git remote server, you don’t need to supply your username and password each time. Read more about SSH keys, how to generate them, and how to add them to your git provider here: [GitHub](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh) and [GitLab](https://docs.gitlab.com/ee/ssh/).
```

File: website/docs/docs/building-a-dbt-project/package-management.md [hunk 3]
```diff
@@ -145,8 +145,10 @@ As of v0.14.0, dbt will warn you if you install a package using the `git` syntax
 
 ### Private packages
 
-#### SSH Key Method
-Private packages can be cloned via SSH and an SSH key. When you use SSH keys to authenticate to your git remote server, you don’t need to supply your username and password each time. Read more about SSH keys, how to generate them, and how to add them to your git provider here: [Github](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh) and [GitLab](https://docs.gitlab.com/ee/ssh/).
+#### SSH Key Method (Command Line only)
+If you're using the Command Line, private packages can be cloned via SSH and an SSH key.
```
Comment (runleonarun) [inline website/docs/docs/building-a-dbt-project/package-management.md:149]:
```suggestion
If you're using the command line interface (CLI), private packages can be cloned via SSH and an SSH key. 
```

File: website/docs/docs/building-a-dbt-project/package-management.md [hunk 4]
```diff
@@ -145,8 +145,10 @@ As of v0.14.0, dbt will warn you if you install a package using the `git` syntax
 
 ### Private packages
 
-#### SSH Key Method
-Private packages can be cloned via SSH and an SSH key. When you use SSH keys to authenticate to your git remote server, you don’t need to supply your username and password each time. Read more about SSH keys, how to generate them, and how to add them to your git provider here: [Github](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh) and [GitLab](https://docs.gitlab.com/ee/ssh/).
+#### SSH Key Method (Command Line only)
```
Comment (runleonarun) [inline website/docs/docs/building-a-dbt-project/package-management.md:148]:
```suggestion
#### SSH Key Method (command line only)
```

**Review by runleonarun — APPROVED** [review]
@mirnawong1 and @jeremyyeo This looks good to me once the suggestions are added, but we might want Naveen to look at it.

**Review by schottj — APPROVED** [review]
thank you for adding this!

File: website/docs/docs/building-a-dbt-project/package-management.md (hunk 4 above)
Comment (mirnawong1) [reply website/docs/docs/building-a-dbt-project/package-management.md:148]: thank you all!

**Pull request merged**